src/napari_vedo_bridge/_cutter_widget.py

```diff
@@ -69,6 +74,24 @@ def send_to_napari(self):
 
         self.napari_viewer.add_surface(mesh_tuple)
 
+    def box_cutter_tool(self):
+        """
+        Add a box cutter tool to the vedo plotter
+        """
+        self.vedo_message.text(
+            "Press r to reset the cutting box\n"
+            'Press spacebar to toggle the cutting box on/off\n'
+            'Press i to invert the selection\n',
+        )
+        if self.pushButton_box_cutter.isChecked():
+            self.box_cutter_widget = BoxCutter(
```

Comment:
Could we - at this point - just do `self.cutter_widget = BoxCutter(...)`? We will only ever have one widget active at a time and we would only have to keep track of the currently active cutting tool.

Reply:
in principle one could have more than one at the same time..